javadoc task fails with unexpected text.

The `javadoc` Gradle task fails with the following message:
<pre>
/Users/rin/Development/Java/Libs/JavaClearHttp/src/main/java/com/levelrin/javaclearhttp/record/Record.java:22: error: unexpected text
 * The {@link this#toString()} method can be useful for viewing all information at once.
       ^
</pre>

--- a/src/main/java/com/levelrin/javaclearhttp/body/Body.java
+++ b/src/main/java/com/levelrin/javaclearhttp/body/Body.java
@@ -42,8 +42,8 @@ public final class Body implements BodyType {
     /**
      * Secondary constructor.
      * This object will use {@link SocketConnection}.
-     * @param map Info at {@link this#map}.
-     * @param headers Info at {@link this#headers}.
+     * @param map Info at {@link Body#map}.
+     * @param headers Info at {@link Body#headers}.
      */
     public Body(final TraceableMap map, final Map<String, String> headers) {
         this(map, headers, new SocketConnection(map));
@@ -52,9 +52,9 @@ public final class Body implements BodyType {
     /**
      * Primary constructor.
      * Usually, it's for testing.
-     * @param map Info at {@link this#map}.
-     * @param headers Info at {@link this#headers}.
-     * @param connection Info at {@link this#connection}.
+     * @param map Info at {@link Body#map}.
+     * @param headers Info at {@link Body#headers}.
+     * @param connection Info at {@link Body#connection}.
      */
     public Body(final TraceableMap map, final Map<String, String> headers, final ConnectionType connection) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/header/Header.java
+++ b/src/main/java/com/levelrin/javaclearhttp/header/Header.java
@@ -48,8 +48,8 @@ public final class Header implements HeaderType {
     /**
      * Secondary constructor.
      * This object will use {@link SocketConnection}.
-     * @param map Info at {@link this#map}.
-     * @param headers Info at {@link this#headers}.
+     * @param map Info at {@link Header#map}.
+     * @param headers Info at {@link Header#headers}.
      */
     public Header(final TraceableMap map, final Map<String, String> headers) {
         this(map, headers, new SocketConnection(map));
@@ -58,9 +58,9 @@ public final class Header implements HeaderType {
     /**
      * Primary constructor.
      * Usually, it's for testing.
-     * @param map Info at {@link this#map}.
-     * @param headers Info at {@link this#headers}.
-     * @param connection Info at {@link this#connection}.
+     * @param map Info at {@link Header#map}.
+     * @param headers Info at {@link Header#headers}.
+     * @param connection Info at {@link Header#connection}.
      */
     public Header(final TraceableMap map, final Map<String, String> headers, final ConnectionType connection) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/http/Http.java
+++ b/src/main/java/com/levelrin/javaclearhttp/http/Http.java
@@ -41,7 +41,7 @@ public final class Http implements HttpType {
 
     /**
      * Primary constructor.
-     * @param url Info at {@link this#url}.
+     * @param url Info at {@link Http#url}.
      */
     public Http(final URL url) {
         this.url = url;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/Headers.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/Headers.java
@@ -38,7 +38,7 @@ public final class Headers {
 
     /**
      * Primary constructor.
-     * @param messages Info at {@link this#messages}.
+     * @param messages Info at {@link Headers#messages}.
      */
     public Headers(final List<String> messages) {
         this.messages = messages;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/Messages.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/Messages.java
@@ -42,7 +42,7 @@ public final class Messages {
     /**
      * Secondary constructor.
      * It can be used when default headers are enough for a HTTP request.
-     * @param map Info at {@link this#map}.
+     * @param map Info at {@link Messages#map}.
      */
     public Messages(final TraceableMap map) {
         this(map, new HashMap<>());
@@ -50,8 +50,8 @@ public final class Messages {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
-     * @param headers Info at {@link this#headers}.
+     * @param map Info at {@link Messages#map}.
+     * @param headers Info at {@link Messages#headers}.
      */
     public Messages(final TraceableMap map, final Map<String, String> headers) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/Port.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/Port.java
@@ -20,7 +20,7 @@ public final class Port {
 
     /**
      * Primary constructor.
-     * @param protocol Info at {@link this#protocol}.
+     * @param protocol Info at {@link Port#protocol}.
      */
     public Port(final String protocol) {
         this.protocol = protocol;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/connection/ConnectionError.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/connection/ConnectionError.java
@@ -37,8 +37,8 @@ public final class ConnectionError {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
-     * @param messages Info at {@link this#messages}.
+     * @param map Info at {@link ConnectionError#map}.
+     * @param messages Info at {@link ConnectionError#messages}.
      */
     public ConnectionError(final TraceableMap map, final List<String> messages) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/connection/HttpSocket.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/connection/HttpSocket.java
@@ -35,7 +35,7 @@ public final class HttpSocket implements ConnectionType {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
+     * @param map Info at {@link HttpSocket#map}.
      */
     public HttpSocket(final TraceableMap map) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/connection/HttpsSocket.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/connection/HttpsSocket.java
@@ -37,7 +37,7 @@ public final class HttpsSocket implements ConnectionType {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
+     * @param map Info at {@link HttpsSocket#map}.
      */
     public HttpsSocket(final TraceableMap map) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/internal/connection/SocketConnection.java
+++ b/src/main/java/com/levelrin/javaclearhttp/internal/connection/SocketConnection.java
@@ -30,13 +30,13 @@ public final class SocketConnection implements ConnectionType {
 
     /**
      * It's responsible for making HTTPS connection.
-     * This will be used when the protocol is HTTPS in the {@link this#map}.
+     * This will be used when the protocol is HTTPS in the {@link SocketConnection#map}.
      */
     private final ConnectionType httpsSocket;
 
     /**
      * Secondary constructor.
-     * @param map Info at {@link this#map}.
+     * @param map Info at {@link SocketConnection#map}.
      */
     public SocketConnection(final TraceableMap map) {
         this(map, new HttpSocket(map), new HttpsSocket(map));
@@ -44,9 +44,9 @@ public final class SocketConnection implements ConnectionType {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
-     * @param httpSocket Info at {@link this#httpSocket}.
-     * @param httpsSocket Info at {@link this#httpsSocket}.
+     * @param map Info at {@link SocketConnection#map}.
+     * @param httpSocket Info at {@link SocketConnection#httpSocket}.
+     * @param httpsSocket Info at {@link SocketConnection#httpsSocket}.
      */
     public SocketConnection(final TraceableMap map, final ConnectionType httpSocket, final ConnectionType httpsSocket) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/method/Method.java
+++ b/src/main/java/com/levelrin/javaclearhttp/method/Method.java
@@ -41,7 +41,7 @@ public final class Method implements MethodType {
     /**
      * Secondary constructor.
      * This object will use {@link SocketConnection}.
-     * @param map Info at {@link this#map}.
+     * @param map Info at {@link Method#map}.
      */
     public Method(final TraceableMap map) {
         this(map, new SocketConnection(map));
@@ -50,8 +50,8 @@ public final class Method implements MethodType {
     /**
      * Primary constructor.
      * Usually, it's for testing.
-     * @param map Info at {@link this#map}.
-     * @param connection Info at {@link this#connection}.
+     * @param map Info at {@link Method#map}.
+     * @param connection Info at {@link Method#connection}.
      */
     public Method(final TraceableMap map, final ConnectionType connection) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/record/Record.java
+++ b/src/main/java/com/levelrin/javaclearhttp/record/Record.java
@@ -19,7 +19,7 @@ import java.util.StringJoiner;
 
 /**
  * It's responsible for providing the request and response records.
- * The {@link this#toString()} method can be useful for viewing all information at once.
+ * The {@link Record#toString()} method can be useful for viewing all information at once.
  */
 public final class Record implements RecordType {
 
@@ -42,9 +42,9 @@ public final class Record implements RecordType {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
-     * @param messages Info at {@link this#messages}.
-     * @param replies Info at {@link this#replies}.
+     * @param map Info at {@link Record#map}.
+     * @param messages Info at {@link Record#messages}.
+     * @param replies Info at {@link Record#replies}.
      */
     public Record(final TraceableMap map, final List<String> messages, final List<String> replies) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/record/request/RequestRecord.java
+++ b/src/main/java/com/levelrin/javaclearhttp/record/request/RequestRecord.java
@@ -32,8 +32,8 @@ public final class RequestRecord implements RequestRecordType {
 
     /**
      * Primary constructor.
-     * @param map Info at {@link this#map}.
-     * @param messages Info at {@link this#rawMessages}.
+     * @param map Info at {@link RequestRecord#map}.
+     * @param messages Info at {@link RequestRecord#rawMessages}.
      */
     public RequestRecord(final TraceableMap map, final List<String> messages) {
         this.map = map;

--- a/src/main/java/com/levelrin/javaclearhttp/record/response/ResponseRecord.java
+++ b/src/main/java/com/levelrin/javaclearhttp/record/response/ResponseRecord.java
@@ -25,7 +25,7 @@ public final class ResponseRecord implements ResponseRecordType {
 
     /**
      * Primary constructor.
-     * @param replies Info at {@link this#rawReplies}.
+     * @param replies Info at {@link ResponseRecord#rawReplies}.
      */
     public ResponseRecord(final List<String> replies) {
         this.rawReplies = replies;

--- a/src/main/java/com/levelrin/javaclearhttp/record/response/Status.java
+++ b/src/main/java/com/levelrin/javaclearhttp/record/response/Status.java
@@ -20,7 +20,7 @@ public final class Status implements StatusType {
 
     /**
      * Primary constructor.
-     * @param statusLine Info at {@link this#statusLine}.
+     * @param statusLine Info at {@link Status#statusLine}.
      */
     public Status(final String statusLine) {
         this.statusLine = statusLine;

